include/octopos.h

```diff
@@ -143,7 +143,7 @@ class octopOS {
     /*! The ids of the used semaphores */
     static std::vector<int> semids;
     /*! The global list of tentacles used for communication with children */
-    static std::vector<std::shared_ptr<tentacle>> tentacles;
+    static std::vector<tentacle*> tentacles;
```

Comment:
Well I like smart pointers. :(

Reply:
I forget why we wanted to remove them. Probably had to do with shm